x-pack/test/security_solution_endpoint/services/endpoint.ts

```diff
@@ -137,13 +139,45 @@ export class EndpointTestResources extends FtrService {
     return deleteIndexedHostsAndAlerts(this.esClient as Client, this.kbnClient, indexedData);
   }
 
+  private async waitForIndex(
```

Comment:
Nice. this is a good generic method that can be used to check for data in any index